.idea/gradle.xml

```diff
@@ -1,11 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="useAutoImport" value="true" />
```

Comment:
Not sure if the changes in `.idea` folder are intended but usually these files are not modified. For example this one could indicate that gradle auto import feature will be disabled in all developer's environments. I think it's safer to add `.idea` to `.gitignore` in general but that's another discussion.

Reply:
I agree that the files under `.idea` should be out of the repo. Anyway, I reverted the line added to this file by IntelliJ.